tests/conftest.py

```diff
@@ -255,8 +255,36 @@ def tiny_config(tmp_path):
         },
     }
 
+
+def _write_config_file(cfg, tmp_path):
+    """Write config file to temp directory"""
     cfg_file = tmp_path / "config.yml"
     with cfg_file.open("w+") as out_file:
         yaml.dump(cfg, out_file)
 
     return cfg_file
+
+
+@pytest.fixture
+def tiny_config(tmp_path):
+    """A config file for a tiny model."""
+    cfg = _get_default_config(tmp_path)
+    return _write_config_file(cfg, tmp_path)
+
+
+@pytest.fixture
+def tiny_config_interval_greater(tmp_path):
```

Comment:
**suggestion:** Rather than creating multiple fixtures that will only be used for a single test (which kind of contradicts the benefit of fixtures), I'd just modify the config for that test specifically. See e.g. [here](https://github.com/Noble-Lab/casanovo/blob/313-save-final-model/tests/unit_tests/test_config.py#L18).